packages/renderer/src/lib/preferences/PreferencesRegistriesEditing.svelte

```diff
@@ -243,6 +243,19 @@ const processPasswordElement = (node: HTMLInputElement, registry: containerDeskt
 </script>
 
 <SettingsPage title="Registries">
+  <svelte:fragment slot="actions">
```

Comment:
do we need a svelte fragment to not break flow layout or just a div would be good ?

Reply:
A div would work fine, I was just running with the 'don't add an extra html component if it's not necessary'. You'd prefer I switch?

Reply:
I think a div would be simpler but I don't have a strong opinion.

Reply:
Switched, and turned on auto-merge.